library/libraries.js

```diff
@@ -1797,6 +1797,26 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             if (win.__GUESS__ && win.__GUESS__.guess) {
                 return { version: UNKNOWN_VERSION };
             }
+            return false;
+        }
+    },
+    'October CMS': {
+        id: 'octobercms',
+        icon: 'octobercms',
+        url: 'https://octobercms.com/',
+        npm: null,
+        test: function (win) {
+            const generatorMeta = document.querySelector('meta[name="generator"][content^="OctoberCMS"]');
+
+            // October CMS resource patterns / paths - search in link, style or script tags
+            const resourcesOctober = /\/modules\/system\/assets\/(css|js)\/framework\.(extras|combined)-min/;
```

Comment:
just including framework.js is pretty common, it should be included in addition to the minified versions.

Reply:
Probably an even better test is to simply test to see if $.request() is available

Reply:
@LukeTowers 

Here's a picture of the regex (to make it easier to view):

![image](https://user-images.githubusercontent.com/57409060/99877439-2f460800-2bf6-11eb-93b9-41b5003d8120.png)

Other cms use `framework.js` and `framework.css` but their `framework` files don't sit in a folder location of `/modules/system/assets/` so that's why I included the folders in the URI's.

![image](https://user-images.githubusercontent.com/57409060/99877702-e55e2180-2bf7-11eb-9096-ecbb4926dbf8.png)

Looking for `$.request()` doesn't seem to work as I did a search in a code search engine and brings up other cms.

Let me know what you think, when you got some free time.

Reply:
Spot a mistake with my regex, github is a pain in the ass with it's heavy caching! Going to copy n paste the files here:

JS Framework files:

https://github.com/octobercms/october/tree/develop/modules/system/assets/js

![image](https://user-images.githubusercontent.com/57409060/99880081-1135d300-2c09-11eb-8ad5-b14ddda8a9af.png)

CSS Framework files:

https://github.com/octobercms/october/tree/develop/modules/system/assets/css

![image](https://user-images.githubusercontent.com/57409060/99880092-2c084780-2c09-11eb-9ada-5c77e5e90b11.png)

So my regex should be like this:

![image](https://user-images.githubusercontent.com/57409060/99880176-b81a6f00-2c09-11eb-9a18-5f8698ee3931.png)

```
/\/modules\/system\/assets\/(css|js)\/framework(\.extras|\.combined)?(-min)?/
```

Gives the results:

![image](https://user-images.githubusercontent.com/57409060/99880240-2e1ed600-2c0a-11eb-80d6-422f3d41313b.png)

Reply:
Looks good to me